pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
Yeah, I'd rather be painfully obvious in the message than ambiguous. Ping me when you've updated this @jayvdb and I'll merge this for @bitglue